_includes/footer.html

```diff
@@ -7,12 +7,18 @@
       <a href="https://www.digitalocean.com">DigitalOcean</a> for sponsoring the
       VMs used in both tracks of the DeCal
     </p>
-
+    <p>
+      <a href="https://www.linode.com">
+        <img src="/images/linode.png" style="height: 50px; width: 125px;" /></a>
```

Comment:
I know we're not 100% consistent about this, but please try to use relative URLs. E.g. `src="{{ "/images/linode.png" | relative_url }}"`

Reply:
Sorry im not too sure what you mean, isn't the URL already relative?

Reply:
If it starts with a slash it's not relative. The idea is that you should be able to host the jekyll site in a subdirectory and `relative_url` will ensure the subdirectory is included in the path.

Reply:
Got it, fixed it.